detox/ios/Detox/Invocation/Action.swift

```diff
@@ -213,14 +213,14 @@ class LongPressAction : Action {
 		
 		let normalizedTargetingPoint = getNormalizedPoint(xPosition: parameters[3], yPosition: parameters[4])
 		
-		var speed = CGFloat(1.0)
+		var speed = CGFloat(0.5)
```

Comment:
Why? It's too slow.

Reply:
It was actually extremely fast in the simulator, with this speed it looks much better